### ISSUE TITLE
Honeywell round thermostat + internet gateway RFG100 replaced by ramses_cc

### ISSUE THREAD

**Issue by renini** [zxdavb/ramses_cc#206]
Hi there, first of all many thanks for this project!

I'm trying to get rid of my RFG100, but still be able to read out the Honeywell Round thermostat temperature and configure a new setpoint from HA.

In my setup there is no controller or anything else, its only:
- intergas openterm heater
- honeywell round thermostat with RF, on ground floor, wired to intergas heater to control first loop and via RF to RFG100 internet gateway
- honeywell round thermostat without RF, on first floor directly wired to intergas heater to control second loop

Obviously nothing can be seen by ramses_rf for the second thermostat, as its only wired up to the heater.
Between the RF round thermostaat and the RFG100 to the thermostat is see the following messages:
- zone_mode
- opentherm_msg
- datetime
- setpoint

And then there also seem to be some broadcast? messages from the RF round:
- actuator_state
- opentherm_sync
- temperature
- relay_demand

And one from the RFG100:
- zone_name

Is it even possible with this setup to be able to do what i want?
And if so, should i try to fake/emulate the RFG100? Could i get it so, that the round still think that its connected to a RFG (icon on the round still shows connected).
From the round i only managed to get the temperature reading as a sensor in HA, not as thermostate to also control the setpoint.
Also have no idea on how to get all the opentherm_msges into sensors.

I really tried all from the wiki/docs, but i think because i have no real controller its maybe some weird setup, where its not possible to do what i want?

TLDR: try to get rid of the RFG100 and the mandatory internet connection, want to control and readout the round thermostate from HA.

**Comment by zxdavb** [comment]
> Is it even possible with this setup to be able to do what i want?

Yes - but difficult... This is not an issue - please see how you get on in the forums:
